plugins/woocommerce/includes/tracks/events/class-wc-products-tracking.php

```diff
@@ -138,13 +138,28 @@ public function track_product_updated_client_side( $post ) {
 						return;
 					}
 
+					var tagsText = $( '[name=\"tax_input[product_tag]\"]' ).val();
 					var currentStockValue = $( '#_stock' ).val();
 					var properties = {
+						attributes:				$( '.woocommerce_attribute' ).length,
+						categories:				$( '[name=\"tax_input[product_cat][]\"]:checked' ).length,
+						'cross-sells':			$( '#crosssell_ids option' ).length ? 'Yes' : 'No',
+						description:			$( '#content' ).val().length ? 'Yes' : 'No',
```

Comment:
This actually doesn't seem to work well when I'm using the "visual" tab on the editor.  If I remove all the content on a saved product, it still shows "Yes."  The "text" tab does work as expected though.

Reply:
Good eyes! I fixed this bug in commit 2dfdf93.